### PR TITLE
feature: custom certificate support

### PR DESCRIPTION
**Pull request by pbertera**
This PR implements the `--sslCert` and `--sslKey` command line options allowing to define a custom certificate for Quay

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -142,13 +156,28 @@ func install() {
 		askBecomePassFlag = "-K"
 	}
 
+	// Set the SSL flag if cert and key are defined
+	var sslCertKeyFlag string
+	if sslCert != "" && sslKey != "" {
+	sslCertAbs, err := filepath.Abs(sslCert)
```
Comment (HammerMeetNail) [inline cmd/install.go:162]: Indentation looks a little off here. Would you mind fixing before merge?

File: cmd/install.go [hunk 2]
```diff
@@ -142,13 +156,28 @@ func install() {
...
+	}
+
 	// Run playbook
 	log.Printf("Running install playbook. This may take some time. To see playbook output run the installer with -v (verbose) flag.")
 	podmanCmd := fmt.Sprintf(`sudo podman run `+
 		`--rm --interactive --tty `+
 		`--workdir /runner/project `+
 		`--net host `+
 		imageArchiveMountFlag+ // optional image archive flag
+		sslCertKeyFlag + // optional ssl cert/key flag
```
Comment (HammerMeetNail) [inline cmd/install.go:180]: Similar request here, for consistency can the whitespace between `sslCertKeyFlag` and the `+` be removed?

**Review by HammerMeetNail — CHANGES_REQUESTED** [review]
@pbertera Thanks for submitting this. I think the PR looks good, but would like to run it through CI. I've updated the target branch to be `quay:PROJQUAY-2247`, once we merge this it should trigger CI. If CI looks good, we can merge to `quay:master`. 

There were a couple formatting things I saw. Would you mind addressing those and rebasing or squashing the commits and then we'll merge to CI?

**Comment by pbertera** [comment]
@HammerMeetNail thanks for the review and for the catch, I pushed the requested changes

**Pull request merged**